### DISCUSSION TITLE
telnet towel.blinkenlights.nl from MS-DOS 6.22

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1228] [Q&A] [answered]
I have a SLiRP Realtek RTL8029AS network card with TCP/IP installed; however, it only works through my Windows for Workgroups 3.11. I can access the telnet and the command towel.blinkenlights.nl, but I do not know how to do it in ms-dos 6.22; how do I do it?

**Comment by ghost** [comment, marked as answer]
You can't unless you have an MS-DOS telnet client installed.

This is not a bug with the emulator.

**Comment by jadasse** [comment]
What client do you recommend?

**Comment by ghost** [comment]
Don't know, never used one.

**Comment by jadasse** [comment]
Oh. Well, i found mtcp. Is that good enough?

**Comment by ghost** [comment]
Again, never tested any. Please lock this issue and if you have problems with it its probably not an emulator issue.

**Comment by jadasse** [comment]
Ok
